frontend/src/style.css

```diff
@@ -133,7 +133,7 @@ img, video {
   }
 
   .page-responsive-width {
-    @apply 3xl:w-[1536px] 2xl:w-[1280px] xl:w-[1024px] lg:w-[768px] md:w-[640px] sm:w-auto m-auto;
+    @apply 3xl:w-[1536px] 2xl:w-[1280px] xl:w-[1024px] lg:w-[768px] md:w-[640px] sm:w-auto mx-auto;
```

Comment:
👍

Reply:
@zhendi can you please help to fix the "sm:w-auto" to "sm:w-full" also

Reply:
@hiveer fixed